source/_integrations/reolink.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: Reolink NVR/camera
+description: Instructions on how to integrate Reolink devices (NVR/cameras) into Home Assistant.
+ha_category:
+  - Camera
+ha_iot_class: Local Polling
+ha_release: 2023.1
+ha_domain: reolink
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+ha_platforms:
+  - Camera
+ha_integration_type: integration
+---
+
+The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
+
+{% include integrations/config_flow.md %}
+
+## Camera streams
+
+This integration creates a few camera entities, one for each stream type with different resolutions: Main, Sub, Ext, and Images.
+The Sub stream camera entity is enabled by default; the other streams are disabled by default.
+The Images stream provides a sequence of image snapshots giving very low latency at the cost of a very low frame rate; this can be used when the hi-res RTMP/RTSP video stream has too much lag.
+
+## Configuration options
+
+In the options menu, the `Protocol` can be configured to RTSP or RTMP streaming protocol. 
+
+## Tested models
+
+The following models have been tested and confirmed to work:
+
+- RLN8-410 NVR
+- RLC-811A
+- Reolink Video Doorbell PoE
+- Reolink Video Doorbell WiFi
+
+Battery-powered cameras are not yet supported.
+
+## Reolink firmware limitations
+
+- The Reolink NVR only sends ONVIF event-notifications if motion happens on the camera connected to the first (index "0") channel.
```

Comment:
What does that mean? (as in, why should the average joe care about this?)

Reply:
The motion/vehicle/pet/person/visitor binary sensors will only work for the camera connected to the first port of the NVR. The binary sensors of the other camera's will only be triggered and updated if the camera connected to port 0 sees motion.

I know, it is stupid and weird, but Reolink is working on the problem.

Now that I type this, this initial basis does not include the binary sensors yet (since I removed them), but that will be the next PR in the comming days.